### PR TITLE
Change to eslint

### PR DESCRIPTION
**Pull request by megawac**

### CODE REVIEW

**Comment by jmeas** [comment]
Dooope. I'll review this today!

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
+    "quotes": [2, "single"]
```
Comment (jmeas) [inline .eslintrc]: What does the "2" mean? I looked in [the docs](http://eslint.org/docs/rules/quotes), but it didn't seem to explain it (it just used it in the example). Should we also include `"avoid-escape"`? That seems like a pretty reasonable option.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
```
Comment (jmeas) [inline .eslintrc]: [the docs](http://eslint.org/docs/rules/strict) say that this property only accepts string values

Comment (jmeas) [inline .eslintrc]: tho' I guess this value does appear in [the babel-eslint readme](https://github.com/babel/babel-eslint#setup). hmmm.

Comment (megawac) [inline .eslintrc]: Think thats fine

File: .eslintrc (hunk 1 above)
Comment (megawac) [inline .eslintrc]: Indentation level, I'd be fine with dropping it

Comment (jmeas) [inline .eslintrc]: JSCS already checks for quotes. Does ESlint make it so we no longer need JSCS? Or should we keep both and only use JSCS for code style preferences, and ESlint for functionality-based preferences?

Comment (jmeas) [inline .eslintrc]: o i c overriding defaults.

**Pull request merged**